src/components/Wallet/WalletSelection.tsx

```diff
@@ -0,0 +1,182 @@
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React, { useState } from 'react';
+import { FlatList, TouchableOpacity, StyleSheet } from 'react-native';
+import { useNavigation } from 'react-navigation-hooks';
+import { BigNumber } from 'bignumber.js';
+
+// Hooks
+import { useStableAssets, useNonStableAssets } from 'hooks/assets';
+import { useChainsConfig } from 'utils/uiConfig';
+
+// Components
+import ButtonGroup from 'components/layout/ButtonGroup';
+import { Spacing } from 'components/legacy/Layout';
+import TokenListItem from 'components/lists/TokenListItem';
+import Icon from 'components/core/Icon';
+import HorizontalProgressBar from 'components/Progress/HorizontalProgressBar';
+import ChainListItem from 'screens/Home/components/ChainListItem';
+
+// Constants
+import { TOKENS, STABLES } from 'constants/walletConstants';
+import { ASSET } from 'constants/navigationConstants';
+
+// Utils
+import { useThemeColors } from 'utils/themes';
+import { wrapBigNumberOrNil } from 'utils/bigNumber';
+import { buildAssetDataNavigationParam } from 'screens/Assets/utils';
+import { formatFiatValue } from 'utils/format';
+
+// Selector
+import { useSupportedChains } from 'selectors/chains';
+import { useFiatCurrency } from 'selectors';
+
+// Types
+import type { AssetCategoryRecordKeys } from 'models/AssetCategory';
+import type { TotalBalances } from 'models/TotalBalances';
+import type { Chain } from 'models/Chain';
+
+type Props = {
+  category: AssetCategoryRecordKeys;
+  accountTotalBalances: TotalBalances;
+  visibleBalance: boolean;
+};
+
+export default function ({ category, accountTotalBalances, visibleBalance }: Props) {
+  const [tabIndex, setTabIndex] = React.useState(0);
+  const [showMore, setShowMore] = useState(false);
+  const { tokens } = useStableAssets();
+  const chains = useSupportedChains();
+  const navigation = useNavigation();
+  const fiatCurrency = useFiatCurrency();
+
+  const chainsConfig = useChainsConfig();
+
+  const { tokens: nonStableTokens, percentage, totalPercentage } = useNonStableAssets();
+
+  const colors = useThemeColors();
+  let listOfAssets = tabIndex === 0 ? nonStableTokens : tokens;
+
+  const items = [
+    { key: TOKENS, title: TOKENS, component: null, color: colors.primaryAccent250 },
+    { key: STABLES, title: STABLES, component: null, color: colors.synthetic180 },
+  ];
+
+  const renderItem = (item: any) => {
+    return (
+      <TokenListItem
+        chain={item.chain}
+        address={item.address}
+        symbol={item.symbol}
+        name={item.name}
+        iconUrl={item.iconUrl}
+        balance={wrapBigNumberOrNil(item.balance?.balance)}
+        style={{ paddingLeft: 0, paddingRight: 0 }}
+        onPress={() => {
+          onNavigate(item, item.chain);
+          // const assetData = buildAssetDataNavigationParam(item, item.chain);
+          // assetData.backDashboad = true;
+          // navigation.navigate(ASSET, { assetData });
```

Comment:
Could we remove dead code please and inline the `onNavigate` function call

Reply:
Fixed